graphql/schema/types/gallery.graphql

```diff
@@ -9,6 +9,10 @@ type Gallery {
   details: String
   rating: Int
   organized: Boolean!
+  created_at: Time!
+  updated_at: Time!
+  file_mod_time: Time!
```

Comment:
Some galleries don't have a `file_mod_time` and return nil from the resolver. The graphql displays an error on these objects as a result.